reproducible-build/measure.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+source /build/srcs/poky/oe-init-build-env
+
+for image in /artifacts/*
+do
+	image_name=$(basename $image | sed -e "s|\..*||")
+	output_file=measurement-${image_name}.json
+	/app/measured-boot $image /artifacts/measurements/$output_file
+done
```

Comment:
do we need to run the mesaured-boot script on all the generated images or rather the one with the .vhd ending. Not sure if it would work with the other types, like qcow2.
Keep in mind that the content in the artifacts directory is not only the images but rather a bunch of other artifacts. So probably, we should restrict the iteration to only those name that starts with the prefix of the image name instead and potentially filter out the extension that we expect to use for the measurement. (e.g. .vhd)

Reply:
Ah makes sense